[2021-11-15 16:44:39] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:8080 时出现错误: Post "http://127.0.0.1:8080": dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-15 16:44:39] [WARNING]: 上报Event数据 {"anonymous":null,"font":0,"group_id":913941037,"message":"现金包吗","message_id":1946145990,"message_seq":1333543,"message_type":"group","post_type":"message","raw_message":"现金包吗","self_id":501273515,"sender":{"age":0,"area":"","card":"","level":"","nickname":"last order","role":"member","sex":"unknown","title":"","user_id":2810167452},"sub_type":"normal","time":1636965875,"user_id":2810167452}
 到 127.0.0.1:8080 失败: Post "http://127.0.0.1:8080": dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-15 16:44:44] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:8080 时出现错误: Post "http://127.0.0.1:8080": dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-15 16:44:44] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":67,"packet_sent":60,"packet_lost":0,"message_received":1,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":1636965877}},"time":1636965882}
 到 127.0.0.1:8080 失败: Post "http://127.0.0.1:8080": dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-15 16:44:49] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:8080 时出现错误: Post "http://127.0.0.1:8080": dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-15 16:44:49] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":67,"packet_sent":60,"packet_lost":0,"message_received":1,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":1636965877}},"time":1636965887}
 到 127.0.0.1:8080 失败: Post "http://127.0.0.1:8080": dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-15 16:44:50] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:8080 时出现错误: Post "http://127.0.0.1:8080": dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-15 16:44:50] [WARNING]: 上报Event数据 {"anonymous":null,"font":0,"group_id":913941037,"message":"为什么会有2个角色5个稿子","message_id":-308592283,"message_seq":1333544,"message_type":"group","post_type":"message","raw_message":"为什么会有2个角色5个稿子","self_id":501273515,"sender":{"age":0,"area":"","card":"","level":"","nickname":"last order","role":"member","sex":"unknown","title":"","user_id":2810167452},"sub_type":"normal","time":1636965888,"user_id":2810167452}
 到 127.0.0.1:8080 失败: Post "http://127.0.0.1:8080": dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-15 16:44:54] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:8080 时出现错误: Post "http://127.0.0.1:8080": dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-15 16:44:54] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":68,"packet_sent":60,"packet_lost":0,"message_received":2,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":1636965888}},"time":1636965892}
 到 127.0.0.1:8080 失败: Post "http://127.0.0.1:8080": dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-15 16:44:59] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:8080 时出现错误: Post "http://127.0.0.1:8080": dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-15 16:44:59] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":68,"packet_sent":60,"packet_lost":0,"message_received":2,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":1636965888}},"time":1636965897}
 到 127.0.0.1:8080 失败: Post "http://127.0.0.1:8080": dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-15 16:45:03] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:8080 时出现错误: Post "http://127.0.0.1:8080": dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-15 16:45:03] [WARNING]: 上报Event数据 {"anonymous":null,"font":0,"group_id":913941037,"message":"pve把","message_id":-1701023245,"message_seq":1333545,"message_type":"group","post_type":"message","raw_message":"pve把","self_id":501273515,"sender":{"age":0,"area":"","card":"","level":"","nickname":"池水河风","role":"member","sex":"unknown","title":"","user_id":1628139954},"sub_type":"normal","time":1636965900,"user_id":1628139954}
 到 127.0.0.1:8080 失败: Post "http://127.0.0.1:8080": dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-15 16:45:03] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:8080 时出现错误: Post "http://127.0.0.1:8080": dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-15 16:45:03] [WARNING]: 上报Event数据 {"duration":60,"group_id":913941037,"notice_type":"group_ban","operator_id":2412125282,"post_type":"notice","self_id":501273515,"sub_type":"ban","time":1636965901,"user_id":1628139954}
 到 127.0.0.1:8080 失败: Post "http://127.0.0.1:8080": dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-15 16:45:04] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:8080 时出现错误: Post "http://127.0.0.1:8080": dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-15 16:45:04] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":71,"packet_sent":62,"packet_lost":0,"message_received":3,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":1636965901}},"time":1636965902}
 到 127.0.0.1:8080 失败: Post "http://127.0.0.1:8080": dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
